src/control/cmd/daos_admin/README.md

```diff
@@ -2,7 +2,7 @@
 
 This page is intended to provide developer-focused documentation for the
 `daos_admin` helper. For installation instructions, please see the
-[DAOS Admin Guide](https://daos-stack.github.io/admin/predeployment_check/#elevated-privileges).
+[DAOS Admin Guide](https://docs.daos.io/latest/admin/predeployment_check/#elevated-privileges).
```

Comment:
Just noticed this link may not be accurate (also applies to the master PR). Is it supposed to point to #privileged-helper ? i.e., this URL instead? https://docs.daos.io/latest/admin/predeployment_check/#privileged-helper

Reply:
fixed